### PR TITLE
First part of restoring CI job, fixes production template deployment …

### PR DESCRIPTION
**Pull request by kevinearls**
This is the first step towards restoring the CI job for testing the Jaeger Openshift templates.  It has been updated to properly deploy the production template after deploying Cassandra separately.  I will still need to update this script to properly do notification.

### CODE REVIEW

**Comment by jkandasa** [comment]
@kevinearls this PR looks good. I see only one concern here. You have removed all the debug options,

example:

old: 
```
stage('all-in-one: clean environment') { 
        steps { 
          sh '''
            echo "Before clean environment:"
            oc get all
            oc delete all,template,daemonset,configmap -l jaeger-infra
            echo "After clean environment:"
            oc get all
          '''
        }
```
new: 
```
 stage('all-in-one: clean environment') {
            steps {
                sh 'oc delete all,template,daemonset,configmap -l jaeger-infra'
            }
        }
```

I see you have removed debug steps from all the stages. I guess with the new approach, it is difficult to rootcase the failures, if any.
@jpkrohling @objectiser @pavolloffay What do you think?

File: jenkins-ci/Jenkinsfile [hunk 1]
```diff
@@ -1,164 +1,96 @@
...
-          oc delete all,template,daemonset,configmap -l jaeger-infra
-          echo "After clean environment:"
-          oc get all
-        '''
-      }
     }
-  }
+}
+
+  /* FIXME restore this
```
Comment (jkandasa) [inline jenkins-ci/Jenkinsfile]: @kevinearls restore this script, to show the detailed status of the build to the community.

File: jenkins-ci/Jenkinsfile [hunk 2]
```diff
@@ -1,164 +1,96 @@
 pipeline {
-  agent any
-    stages {
-      stage('all-in-one: clean environment') { 
-        steps { 
-          sh '''
-            echo "Before clean environment:"
```
Comment (jkandasa) [inline jenkins-ci/Jenkinsfile]: @kevinearls debug steps removed here.

**Comment by objectiser** [comment]
@jkandasa Once we are happy it is working as expected, then I guess the debug info is redundant - and can always be added back in if things seem to be failing.

**Comment by kevinearls** [comment]
@jkandasa I have re-enabled the push-logs script.  Can you approve (and/or merge) this now?

As @objectiser suggested, I removed the debugging info as I thought it was redundant and mostly cluttered up the logs.  If I think we need it in the long term I will add it back.

**Pull request merged**